pytest_factoryboy/plugin.py

```diff
@@ -54,15 +54,15 @@ def get_deps(self, request: FixtureRequest, fixture: str, deps: set[str] | None
                     deps.update(self.get_deps(request, argname, deps))
         return deps
 
-    def get_current_deps(self, request: FixtureRequest) -> set[str]:
+    def get_current_deps(self, request: FixtureRequest | SubRequest) -> set[str]:
```

Comment:
Isn't SubRequest a subclass of FixtureRequest?

Reply:
yes, it is, but we usually are using attributes and methods from the `SubRequest` class that aren't implemented by `FixtureRequest`

Reply:
My point is, we can just annotate this param as `SubRequest`, right?